frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/chat_input/select_sources_menu.dart

```diff
@@ -222,60 +222,50 @@ class _PopoverContent extends StatelessWidget {
     BuildContext context,
     ChatSettingsState state,
   ) {
-    return state.selectedSources.map(
-      (e) => ChatSourceTreeItem(
-        key: ValueKey(
-          'selected_select_sources_tree_item_${e.view.id}',
-        ),
-        chatSource: e,
-        level: 0,
-        isDescendentOfSpace: e.view.isSpace,
-        isSelectedSection: true,
-        onSelected: (chatSource) {
-          context.read<ChatSettingsCubit>().toggleSelectedStatus(chatSource);
-        },
-        height: 30.0,
-        visibilityGetter: (view) {
-          if (view.id == context.read<ChatSettingsCubit>().chatId) {
-            return IgnoreViewType.hide;
-          }
-
-          return view.layout.isDocumentView
-              ? IgnoreViewType.none
-              : IgnoreViewType.disable;
-        },
-      ),
-    );
+    return state.selectedSources
+        .where((e) => e.ignoreStatus != IgnoreViewType.hide)
+        .map(
+          (e) => ChatSourceTreeItem(
+            key: ValueKey(
+              'selected_select_sources_tree_item_${e.view.id}',
+            ),
+            chatSource: e,
+            level: 0,
+            isDescendentOfSpace: e.view.isSpace,
```

Comment:
Redundant since you already pass `e`

Reply:
Do you mean this `isDescendentOfSpace` field?